wasmbinding/test/messages_test.go

```diff
@@ -336,7 +336,7 @@ func TestBurn(t *testing.T) {
 				Amount:          mintAmount,
 				BurnFromAddress: lucky.String(),
 			},
-			expErr: true,
+			expErr: false,
```

Comment:
Not quite understanding this one. Why do we not expect an error when burning from an address that is not the denom admin?

Reply:
```
if burn.BurnFromAddress != "" && burn.BurnFromAddress != contractAddr.String() {
		return wasmvmtypes.InvalidRequest{Err: "BurnFromAddress must be \"\""}
	}
```

Previously, we had a check for dis-allowing admin accounts to burn from all accounts, this change removes that constraint

Reply:
Oh wow I had a fundamental incorrect understanding of what burn from meant. I thought it meant "burn tokens AS this user" not "burn tokens RESIDING in this users account". Thanks for the clarification!